build/docker/centos7/build/Dockerfile

```diff
@@ -32,13 +32,21 @@ RUN rpmkeys --import mono-project.com.rpmkey.pgp && \
         lz4-devel \
         lz4-static \
         mono-devel \
-        rh-python36 \
-        rh-python36-python-devel \
-        rh-ruby26 \
         rpm-build \
         tcl-devel \
         unzip \
         wget && \
+    if [ "$(uname -p)" == "aarch64" ]; then \
```

Comment:
Why do we need to use python3.8 on arm and not x86?

Reply:
python3.6 & ruby2.4 are not available on CentOS7 on Arm.
I prefer to upgrade on Intel as well, but I suppose that will require a bit more testing.
We can address it in a separate PR.